STORACHA.md

```diff
@@ -0,0 +1,173 @@
+# Storacha IPFS Setup and AES-256 Proof Encryption
+
+This guide explains how to set up IPFS uploads via Storacha for the dApp, how to generate and encrypt your Storacha proof with AES-256, and how to configure the required environment variables.
+
+The dApp server route `dapp/src/pages/api/w3up-delegation.js` expects:
+- `STORACHA_SING_PRIVATE_KEY` (required): your W3UP/Storacha ED25519 principal private key.
+- `STORACHA_PROOF` (required): either the plain proof string or a 64-hex key used to decrypt a local `encrypted_proof.bin`.
+
+If `STORACHA_PROOF` is exactly 64 characters, the app treats it as an AES key (hex) and decrypts a local `encrypted_proof.bin` using `dapp/src/utils/decryptAES256.ts`.
+
+---
+
+## 1) Prerequisites
+- Bun installed for dApp development.
+- A Storacha (Web3.Storage W3UP) account and a created Space/Proof.
+- This repository cloned locally.
+
+References in the codebase:
+- Encryption tool: `tools/AES256-encrypt.js`
+- Decryption helper: `dapp/src/utils/decryptAES256.ts`
+- Env typings: `dapp/src/env.d.ts`
+- API usage: `dapp/src/pages/api/w3up-delegation.js`
+
+---
+
+## 2) Create a Storacha Proof
+
+Follow Storacha/W3UP docs to create a Space and obtain its Proof (a UCAN proof string). You will end up with a proof string you can paste into an environment variable or encrypt locally.
+
+Install CLI:
+
+```bash
+npm install -g @web3-storage/w3cli
```

Comment:
This is incorrect as it uses the old package. You even made a PR to update that 😅